themes/custom/ts_wrin/sass/components/_popup_modal.scss

```diff
@@ -2,13 +2,60 @@
   display: none;
 }
 
+.region-top-modals .block-modal {
+  display: none;
+  position: absolute;
```

Comment:
I don't think this needs to be absolute -- the modal can come in from the top and go all the way across the screen.